src/nvim/options.lua

```diff
@@ -2488,12 +2488,6 @@ return {
       varname='p_tbidi',
       defaults={if_true={vi=false}}
     },
-    {
-      full_name='termencoding', abbreviation='tenc',
-      type='string', scope={'global'},
-      vi_def=true,
-      defaults={if_true={vi=""}}
-    },
```

Comment:
we had tentative plans to restore this, or at least detect it internally for converting  input/output to/from the terminal. https://github.com/neovim/neovim/issues/7445

Reply:
Removing it causes problems with plugins, e.g. vim-gitgutter: https://github.com/airblade/vim-gitgutter/pull/671

Reply:
This should likely get handled like with gvim, where it is forced to be "utf-8" always.

Reply:
Yes, that's why we left it in, but I'm not really sure that's a good reason. If we don't support options we should just remove them, generally...

Reply:
> This should likely get handled like with gvim, where it is forced to be "utf-8" always.

I just noticed that as well. Hmm.

Reply:
Restored it in https://github.com/neovim/neovim/pull/11662